al_hostserver/altrain.py

```diff
@@ -284,6 +287,9 @@ def parse_args(argv):
                         help="Specifies the directory of the outer loop repo.")
     parser.add_argument('--outer-loop-url', default=None, dest="outer_loop_url",      metavar="<outer_loop_url>",
                         help="Specifies the URL of a running outer loop server.")
+    parser.add_argument('--debug', action='store_true', help="Makes logging messages visible in the console.")
+    parser.add_argument('--debug-agent', action='store_true', dest="debug_agent", help="Makes logging messages for the agent visible in the console.") 
```

Comment:
I don't think its a good idea to constantly add new flags for different debuggers like this. There are many more instances of the logging module used throughout AL_Core that would not respond to this style of flagging. A more general solution where you provide some kind of delimited list that switches on multiple loggers at once seems like the ideal in this case.